GOVERNANCE.md

```diff
@@ -664,11 +664,11 @@ Team:
 * Documentation team members are given permissions to [pymc-examples](https://github.com/pymc-devs/pymc-examples)
```

Comment:
Community team members now also get github permissions to pymcon and pymc-data-umbrella repos (plus other repos we might create for other events and initiatives)

Reply:
I changed this and the permission examples.  I think I hit got it correct, but some of the examples are a bit complicated.